Instructor-Guide--Assignments--Summary.md

```diff
@@ -43,11 +45,32 @@ To the right of the grade distribution chart is a set of statistics that give a
 
 > :spiral_notepad: **Note:** In brackets, next to each of the statistics that are shown as a fraction, is the same statistic fraction but displayed as a percentage instead.
 
-Finally, at the bottom of the view, below the grade distribution chart, you can see another distribution of grades for an assignment. What differentiates this grade distribution graph from the one above however, is that each bar from the graph above is broken up in order to also show the distribution of marks given by each grader.
+#### Criteria Distribution
+
+Below the grade distribution chart, you can further analyze the distribution of grades using the criteria distribution graph. This graph shows the distribution of marks given for each associated criterion.
+
+![Criterion Grade Distribution Graph](images/criteria-summary-stats-grade-distribution.png)
+
+Each criterion corresponds to a colour shown in the labels just above the graph. By default, all grade data for criterions are hidden. In order to reveal the grade distribution for a specific criterion(s), simply click on the labels of all the criterion you wish to view and compare.
+
+To help you get a quick overview of student performance for each criteria, next to the criteria distribution graph is a table that shows each criterion along with the average grade received for that criterion. Clicking on the drop down arrow next to each criterion will reveal additional summary statistics for that particular criterion.
+
+![Criterion Grade Distribution Table](images/criteria-summary-stats-table.png)
+
+The additional criterion statistics currently shown are:
+
+- **Average:** The average point grade of submissions for the given criterion (including submissions which recieved a zero) out of the maximum possible mark for that criterion.
```

Comment:
`the average point grade of submissions` doesn't sound right to my ear... can we change it to just `the average grade`

Reply:
Do we really have to say `out of the maximum possible mark for that criterion`? That seems to be implied by both the pictures and the way that grades work on MarkUs and elsewhere.

Can we rephrase this whole thing to:

- **Average:** The average grade received for the given criterion
- **Median:** The median grade received for the given criterion
- etc.

And then at the bottom you can just write once: 

`All statistics include submissions which received a zero for the given criterion`

Reply:
Can you also clarify whether this includes incomplete submissions and/or submissions that have not been released to students yet.

Reply:
> Do we really have to say `out of the maximum possible mark for that criterion`? That seems to be implied by both the pictures and the way that grades work on MarkUs and elsewhere.
> 

Not really. This was worded to be consistent with the grade distribution documentation above and to be explicitly clear what the individual fractions shown meant. I'll change it though.